helm_deploy/hmpps-support-additional-needs-ui/values.yaml

```diff
@@ -54,6 +54,8 @@ generic-service:
     hmpps-support-additional-needs-ui-client-creds:
       CLIENT_CREDS_CLIENT_ID: 'CLIENT_CREDS_CLIENT_ID'
       CLIENT_CREDS_CLIENT_SECRET: 'CLIENT_CREDS_CLIENT_SECRET'
+      CURIOUS_CLIENT_ID: "CURIOUS_CLIENT_ID"
```

Comment:
HAAR have created this secret and associated client-creds in dev, so this should deploy to dev OK;
I'll ask them to create the client-creds and secrets etc in preprod and prod next week 👍